docs/AboutUs.md

```diff
@@ -1,52 +1,34 @@
 # About Us
 
 We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
+Our project is a Task Manager for Jim.
 
 ## Project Team
 
-#### [Damith C. Rajapakse](http://www.comp.nus.edu.sg/~damithch) <br>
-<img src="images/DamithRajapakse.jpg" width="150"><br>
-**Role**: Project Advisor
+#### [Ong Li-Jen]() <br>
+<img src="images/" width="150"><br>
```

Comment:
- Please upload a recent photo of yourself.
- Do not put what should have been an issue/task into a commit message (i.e. "... please add your picture to images ...")